instructions/docs/getting-started.md

```diff
@@ -46,7 +46,7 @@ In this exercise, we will walk you through setting up your environment, step by
 ## Explore Further
 
 * Check out the `~/environment/workshop/cdk` directory to see how the workshop resources are described using CDK
-* [Who are Faythe and Walter, anyway?](https://en.wikipedia.org/wiki/Alice_and_Bob#Cast_of_characters) And the other characters you may encounter as you explore.
+* <a href="https://en.wikipedia.org/wiki/Alice_and_Bob#Cast_of_characters" target="_blank">Who are Faythe and Walter?</a> And the other characters you may encounter as you explore.
```

Comment:
Up to you, but I would remove "And the other characters you may encounter as you explore.", it's not a complete sentence and reads a bit weird

Reply:
Yeah, makes sense. I'll fix that. Thanks!